lib/src/data/model/card_model.dart

```diff
@@ -0,0 +1,102 @@
+import '../../domain/entity/card.dart';
+
+class CardModel extends Card {
+  CardModel(
+    String? cardId,
+    String? dbfId,
+    String? name,
+    String? cardSet,
+    String? type,
+    String? faction,
+    String? rarity,
+    int? cost,
+    int? attack,
+    int? health,
+    int? durability,
+    int? armor,
+    String? text,
+    String? inPlayText,
+    String? flavor,
+    String? artist,
+    bool? collectible,
+    bool? elite,
+    String? race,
+    String? playerClass,
+    String? multiClassGroup,
+    String? classes,
+    String? howToGet,
+    String? howToGetGold,
+    String? img,
+    String? imgGold,
+    String? locale,
+    List<String>? mechanics,
+  ) : super(
```

Comment:
You can create this constructor like CardModel({...}), to made named parameters inside.
When you made a new instance of CardModel instead to created like this:
CardModel(
    cardsJson['cardId'],
    ....
)
You are going to create a card like this:
CardModel(
    cardId: cardsJson['cardId'],
    ....
)

Reply:
Done